emcc.py

```diff
@@ -914,13 +892,8 @@ def need_llvm_debug_info(options):
 
     options, settings_changes, newargs = parse_args(newargs)
 
-    if use_cxx:
-      clang_compiler = CXX
-    else:
-      clang_compiler = CC
-
     if '-print-search-dirs' in newargs:
-      return run_process([clang_compiler, '-print-search-dirs'], check=False).returncode
+      return run_process([CC, '-print-search-dirs'], check=False).returncode
```

Comment:
shouldn't this use CXX if we are in C++ mode? (there might be other search dirs in that case?)

Reply:
Apparently this doesn't differ `diff <(clang -print-search-dirs) <(clang++ -print-search-dirs)` is empty as is `diff <(gcc -print-search-dirs) <(g++ -print-search-dirs)`

Reply:
`-print-search-dirs` does not report include paths, for example, which might differ.